test/testenv/cmutil.go

```diff
@@ -199,27 +203,47 @@ func RollHotBuckets(deployment *Deployment) bool {
 	return false
 }
 
-// RollingRestartEndpointResponse is represtentation of /services/cluster/master/info endpiont
-type RollingRestartEndpointResponse struct {
+// ClusterMasterInfoEndpointResponse is represtentation of /services/cluster/manager/info endpoint
+type ClusterMasterInfoEndpointResponse struct {
 	Entry []struct {
 		Content struct {
-			RollingRestartFlag bool `json:"rolling_restart_flag"`
+			RollingRestartFlag bool              `json:"rolling_restart_flag"`
+			ActiveBundle       map[string]string `json:"active_bundle"`
 		} `json:"content"`
 	} `json:"entry"`
 }
 
+// ClusterMasterInfoResponse Get cluster Manager response
+func ClusterMasterInfoResponse(deployment *Deployment, podName string) ClusterMasterInfoEndpointResponse {
```

Comment:
Likewise, if this function is only used within the tests framework, we could replace "Master" with "Manager"